packages/nx/src/commands-runner/command-graph-utils.ts

```diff
@@ -0,0 +1,65 @@
+import { CommandGraph } from './command-graph';
+
+function _findCycle(
```

Comment:
Make the task graph utils typings more generic and use the same functions

Reply:
changed